openapi.yaml

```diff
@@ -502,6 +505,34 @@ components:
           value: "making"
         spinningDyeing:
           value: "spinning,dyeing"
+    dyeingMediumParam:
+      name: dyeingMedium
+      in: query
+      description: |
+        Ce paramètre permet de préciser le support sur lequel s'effectue la teinture ; il peut prendre les valeurs suivantes :
+
+        - `article` lorsque la teinture est effectuée sur l'article ;
+        - `fabric` lorsqu'elle est effectuée sur le tissu ;
+        - `yarn` lorsqu'elle est effectuée sur le fil.
+
+        En l'absence d'utilisation explicite du paramètre, la teinture sur étoffe est implicitement utilisée par défaut.
+      required: false
+      style: form
+      schema:
+        type: string
+      examples:
+        default:
+          summary: "par défaut, sur étoffe"
+          value: ""
+        sur pièce:
+          summary: "sur article"
+          value: "article"
+        fabric:
+          summary: "sur tissu"
```

Comment:
pareil ici, on a "sur tissu" alors que ligne 525 on a "sur étoffe".

Reply:
Adressed in https://github.com/MTES-MCT/ecobalyse/pull/174/commits/c96549a02fb25477323960116be953c8ad0babb4